docs/docs/configuration/overview.md

```diff
@@ -72,144 +72,145 @@ An example [oauth2-proxy.cfg](https://github.com/oauth2-proxy/oauth2-proxy/blob/
+| `--prefer-email-to-user`                   | bool           | Prefer to use the Email address as the Username when passing information to upstream. Will only use Username if Email is unavailable, e.g. htaccess authentication. Used in conjunction with `--pass-basic-auth` and `--pass-user-headers` | false |
+| `--pass-host-header`                       | bool           | pass the request Host Header to upstream | true |
+| `--pass-user-headers`                      | bool           | pass X-Forwarded-User, X-Forwarded-Groups, X-Forwarded-Email and X-Forwarded-Preferred-Username information to upstream | true |
+| `--profile-url`                            | string         | Profile access endpoint | |
+| `--prompt`                                 | string         | [OIDC prompt](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest); if present, `approval-prompt` is ignored | `""` |
+| `--provider`                               | string         | OAuth provider | google |
+| `--provider-ca-file`                       | string \| list |  Paths to CA certificates that should be used when connecting to the provider.  If not specified, the default Go trust sources are used instead. |
+| `--provider-display-name`                  | string         | Override the provider's name with the given string; used for the sign-in page | (depends on provider) |
+| `--ping-path`                              | string         | the ping endpoint that can be used for basic health checks | `"/ping"` |
+| `--ping-user-agent`                        | string         | a User-Agent that can be used for basic health checks | `""` (don't check user agent) |
+| `--ready-path`                             | string         | the ready endpoint that can be used for deep health checks | `"/ready"` |
+| `--metrics-address`                        | string         | the address prometheus metrics will be scraped from | `""` |
+| `--proxy-prefix`                           | string         | the url root path that this proxy should be nested under (e.g. /`<oauth2>/sign_in`) | `"/oauth2"` |
+| `--proxy-websockets`                       | bool           | enables WebSocket proxying | true |
+| `--pubjwk-url`                             | string         | JWK pubkey access endpoint: required by login.gov | |
+| `--real-client-ip-header`                  | string         | Header used to determine the real IP of the client, requires `--reverse-proxy` to be set (one of: X-Forwarded-For, X-Real-IP, or X-ProxyUser-IP) | X-Real-IP |
+| `--redeem-url`                             | string         | Token redemption endpoint | |
+| `--redirect-url`                           | string         | the OAuth Redirect URL, e.g. `"https://internalapp.yourcompany.com/oauth2/callback"` | |
+| `--relative-redirect-url`                  | bool           | allow relative OAuth Redirect URL.` | |
```

Comment:
@axel7083 
You missed a \` probably from the previous line. Talking about the `redirect-url` line, I would love so see a slightly more extensive documentation by adding slight more context for a possible use case or just adding an example url like `/my-oauth2/callback`. Furthermore the line should start with a capital `Allow`. I know this isn't really consistent through the documentation right now but I'm going to fix that soon.